File: Add Code Here/C++/tree/Java/27. RemoveElement.java
class Solution {
    
    void delElement(int[] arr, int key, int len){
        for(int i=key;i<len-1;i++){
            arr[i]=arr[i+1];
        }
    }
    
    
    public int removeElement(int[] nums, int val) {
        int len=nums.length;
        
        for(int i=0 ; i < len ; i++){
            if(nums[i]==val){
                delElement(nums, i, len);
                len--;
                i--;
            }
        }
        return len;
    }
}

File: Add Code Here/LeetCode Problems/27. Remove Element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int n = nums.size();
        for(int i = 0; i < n;i++){
            if(nums[i] == val){
                nums.erase(nums.begin() + i);
                n--;
                i--;
            }
        }
        return n;
        
    }
};
